packages/service/src/utils/db.test.ts

```diff
@@ -0,0 +1,12 @@
+import { db, pool, schema } from "./db";
+
+afterAll(()=>{
+  pool.end();
+})
+
+describe('db', () => {
+  it('retrieves data', async () => {
+    const documents = await db.sql<db.SQL, schema.document.Selectable[]>`SELECT * FROM ${"document"}`.run(pool);
+    expect(documents).toEqual([]);
```

Comment:
Instead of adding a new test lets fill in the test we already have over here:

https://github.com/chewnoill/collaborative-editor/blob/master/packages/service/src/db.test.ts#L11-L12